packages/fluentui/react-northstar/src/themes/teams/components/SvgIcon/svgIconVariables.ts

```diff
@@ -30,7 +30,7 @@ export interface IconVariables {
   sizeModifier?: IconSizeModifier;
 }
 
-export default (siteVars): Partial<IconVariables> => ({
+export default (siteVars): Partial<SvgIconVariables> => ({
```

Comment:
```suggestion
export default (siteVars): SvgIconVariables => ({
```

Should `Partial` be there?

Reply:
No, good call, it was IconVariables previosly